docs/core/tools/global-json.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: global.json overview
 description: Learn how to use the global.json file to set the .NET Core SDK version when running .NET Core CLI commands.
-ms.date: 12/03/2018
+ms.date: 12/15/2019
 ms.custom: "updateeachrelease, seodec18"
 ---
 # global.json overview
 
-[!INCLUDE [topic-appliesto-net-core-all](../../../includes/topic-appliesto-net-core-all.md)]
+**This article applies to: ✓** .NET Core 2.x SDK and later versions
 
 The *global.json* file allows you to define which .NET Core SDK version is used when you run .NET Core CLI commands. Selecting the .NET Core SDK is independent from specifying the runtime your project targets. The .NET Core SDK version indicates which versions of the .NET Core CLI tools are used. In general, you want to use the latest version of the tools, so no *global.json* file is needed.
```

Comment:
Can we call out the last sentence a bit more - perhaps by putting it in a separate paragraph first. 
```suggestion
 In general, you want to use the latest version of the SDK tools, so no *global.json* file is needed. In some advanced scenarios, users wish to control the version of the SDK tools, and this article explains how to do this.
 
 The *global.json* file allows you to define which .NET Core SDK version is used when you run .NET Core CLI commands. Selecting the .NET Core SDK is independent from specifying the runtime your project targets. The .NET Core SDK version indicates which versions of the .NET Core CLI tools are used.
```

Reply:
I'll make some changes to your suggestion here since it mixes you and 3rd person.

Reply:
I'm also thinking we need to define what a global.json file is and what it does before we say what's our recommendation. This change also impacts SEO.